src/main/java/com/spotify/reaper/ReaperApplicationConfiguration.java

```diff
@@ -33,10 +36,12 @@
 
   @JsonProperty
   @NotNull
-  private String repairParallelism;
+  private RepairParallelism repairParallelism;
 
   @JsonProperty
   @NotNull
+  @Min(0)
```

Comment:
We can probably just add and IllegalArgumentException to be thrown in the location where the assertion was used previously.